jmslib/src/main/java/com/redhat/mqe/lib/SenderClient.java

```diff
@@ -150,7 +150,6 @@ public void startClient() {
         } catch (JMSException | IllegalArgumentException jmse) {
             LOG.error("Error while sending a message!", jmse.getMessage());
             jmse.printStackTrace();
-            System.exit(1);
```

Comment:
What is the goal there? This change does not have comment, and the commit message does not explain why this is being done either.

Reply:
Most probably connected to the issue above - in failover scenarios, client should not get closed after JMSException (killed broker for example).

But I don't recall it properly TBH. :-/